test/runtime/Policies.go

```diff
@@ -1015,6 +1032,118 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
+			ExpectWithOffset(1, res.WasSuccessful()).Should(BeFalse(),
+				"unexpectedly able to access %s when access should only be allowed to CIDR", hostIP)
+
+			By(fmt.Sprintf("Pinging %s from %s (shouldn't work)", helpers.App2, helpers.App1))
+			res = vm.ContainerExec(helpers.App1, helpers.Ping(helpers.App2))
+			ExpectWithOffset(1, res.WasSuccessful()).Should(BeFalse(),
+				"expected ping to %s to fail", helpers.App2)
+
+			httpd2, err := vm.ContainerInspectNet(helpers.Httpd2)
+			ExpectWithOffset(1, err).Should(BeNil(),
+				"Unable to get networking information for container %s", helpers.Httpd2)
+
+			By(fmt.Sprintf("Accessing /index.html in %s from %s (shouldn't work)", helpers.App2, helpers.App1))
+			res = vm.ContainerExec(helpers.App1, helpers.CurlFail("%s://%s/index.html", proto, httpd2[helpers.IPv4]))
+			ExpectWithOffset(1, res.WasSuccessful()).Should(BeFalse(),
+				"unexpectedly able to access %s when access should only be allowed to CIDR", helpers.Httpd2)
+		}
+
+		It("Tests egress with CIDR+L4 policy", func() {
+			By(fmt.Sprintf("Importing policy which allows egress to %s from %s", otherHostIP, helpers.App1))
+			policy := fmt.Sprintf(`
+			[{
+				"endpointSelector": {"matchLabels":{"id.%s":""}},
+				"egress": [{
+					"toCIDR": [
+						"%s"
+					],
+					"toPorts": [
+						{"ports":[{"port": "80", "protocol": "TCP"}]}
+					]
+				}]
+			}]`, helpers.App1, otherHostIP)
+
+			testCIDRL4Policy(policy, otherHostIP, "http")
+		})
+
+		It("Tests egress with CIDR+L4 policy to external https service", func() {
+			cloudFlare := "1.1.1.1"
+
+			By(fmt.Sprintf("Importing policy which allows egress to %s from %s", otherHostIP, helpers.App1))
+			policy := fmt.Sprintf(`
+			[{
+				"endpointSelector": {"matchLabels":{"id.%s":""}},
+				"egress": [{
+					"toCIDR": [
+						"%s/30"
+					],
+					"toPorts": [
+						{"ports":[{"port": "443", "protocol": "TCP"}]}
+					]
+				}]
+			}]`, helpers.App1, cloudFlare)
+
+			testCIDRL4Policy(policy, cloudFlare, "https")
+		})
+
+		It("Tests egress with CIDR+L7 policy", func() {
+			By(fmt.Sprintf("Importing policy which allows egress to %s from %s", otherHostIP, helpers.App1))
+			policy := fmt.Sprintf(`
+			[{
+				"endpointSelector": {"matchLabels":{"id.%s":""}},
+				"egress": [{
+					"toCIDR": [
+						"%s/32"
+					],
+					"toPorts": [{
+						"ports":[{"port": "80", "protocol": "TCP"}],
+						"rules": {
+							"HTTP": [{
+							  "method": "GET",
+							  "path": "/index.html"
+							}]
+						}
+					}]
+				}]
+			}]`, helpers.App1, otherHostIP)
+
+			testCIDRL4Policy(policy, otherHostIP, "http")
+
+			By(fmt.Sprintf("Accessing /private on %s from %s should fail", otherHostIP, helpers.App1))
+			res := vm.ContainerExec(helpers.App1, helpers.CurlWithHTTPCode("http://%s/private", otherHostIP))
+			res.ExpectContains("403", "unexpectedly able to access http://%s:80/private when access should only be allowed to /private", otherHostIP)
```

Comment:
This assert message does not make sense to me no? should be: `unexpectedly able to access http://%s:80/private when access should only be allowed to /index.html", otherHostIP` no?